graphics/managed_surface.h

```diff
@@ -306,6 +313,38 @@ class ManagedSurface {
 		return Common::Rect(0, 0, this->w, this->h);
 	}
 
+	/**
+	 * Copy another surface into this one.
+	 */
+	void simpleBlitFrom(const Surface &src, const Palette *srcPalette = nullptr);
```

Comment:
What about naming it `copyBlitFrom()`?

Reply:
Possibly, however it also handles key blits and format conversion, not just copy blits.